lib/experimental/block-editor-settings-mobile.php

```diff
@@ -33,7 +71,71 @@ function gutenberg_get_block_editor_settings_mobile( $settings ) {
 		$settings['__experimentalEnableListBlockV2'] = true;
 	}
 
-	return $settings;
+	return keep_supported_block_editor_settings_mobile(
+		$settings,
+		array(
+			'alignWide'                        => true,
+			'allowedBlockTypes'                => true,
+			'allowedMimeTypes'                 => true,
+			'defaultEditorStyles'              => true,
+			'blockCategories'                  => true,
+			'isRTL'                            => true,
+			'imageDefaultSize'                 => true,
+			'imageDimensions'                  => true,
+			'imageEditing'                     => true,
+			'imageSizes'                       => true,
+			'maxUploadFileSize'                => true,
+			'__unstableGalleryWithImageBlocks' => true,
+			'disableCustomColors'              => true,
+			'disableCustomFontSizes'           => true,
+			'disableCustomGradients'           => true,
+			'disableLayoutStyles'              => true,
+			'enableCustomLineHeight'           => true,
+			'enableCustomSpacing'              => true,
+			'enableCustomUnits'                => true,
+			'colors'                           => true,
+			'fontSizes'                        => true,
+			'__experimentalStyles'             => array(
+				'elements'   => true,
+				'spacing'    => true,
+				'blocks'     => true,
+				'color'      => true,
+				'typography' => true,
+			),
```

Comment:
At the moment, I am not certain what, if any, of `__experimentalStyles` is utilized. However, it appears to be expected in the current test.

https://github.com/WordPress/gutenberg/blob/e3a6917ff0353bba5d40660acb1bc273de2b2541/phpunit/experimental/block-editor-settings-mobile-test.php#L64

Reply:
I'm not entirely sure where this is used, either, by I recall that this is specifically handled (as `rawStyles`) in the app's db, so good idea to add it. Fwiw, I generated the allow-list from an existing response from a specific site, so there may be other things that are missing in this list that would (and possibly should) appear for other site-types.